packages/babel-parser/test/fixtures/typescript/cast/arrow-in-parens-with-parens-node/options.json

```diff
@@ -0,0 +1,4 @@
+{
+  "createParenthesizedExpressions": true,
```

Comment:
should we add this option to `babel-plugin-syntax-typescript`?

Reply:
No this is only for tools using babel-parser directly.
Babel would not understand the AST that gets returned when this option is active.